src/SpecialThanks.scss

```diff
@@ -0,0 +1,25 @@
+@import './allVariables.scss';
+
+.head {
+    margin: 2rem 0px;
+    text-align: center;
+}
+
+.card-container {
+    background: transparent;
+    background-color: transparent;
+    display: grid;
+    grid-auto-flow: column;
+    grid-auto-columns: 21%;
```

Comment:
why 21%, not 20% or 25%?

Reply:
So you can see that there is more content. Any value works which when multiply by some amount would produce 100%.